### PR TITLE
Tags input

### PR DESCRIPTION
**Pull request by thecalcc**

### CODE REVIEW

File: app-typescript/components/TagInput.tsx [hunk 1]
```diff
@@ -1,26 +1,54 @@
...
+            fullWidth={this.props.fullWidth}
+            htmlId={this.htmlId}
+            tabindex={this.props.tabindex}>
+                <Chips
+                    className={`tags-input--multi-select sd-input__input ${this.props.disabled ? ' tags-input__padding-disabled' : ''}`}
+                    allowDuplicate={false}
+                    separator=","
+                    onChange={({value}) => onChange(value)}
+                    value={value}
+                    placeholder={placeholder ? placeholder : 'Type Here'}
```
Comment (tomaskikutis) [inline app-typescript/components/TagInput.tsx]: don't use a default placeholder here - we can't translate strings in ui-framework project

File: examples/pages/react/TagInputDocs.tsx [hunk 2]
```diff
@@ -32,13 +32,17 @@ export default class TagInputDocs extends React.Component<{}, {value: Array<stri
...
+                                error={'Error Message'}
+                                label={'TreeSelect Label'}
                                 onChange={(value) => {
                                     this.setState({
                                         value: value,
                                     });
                                 }}
                                 value={this.state.value}
-                                placeholder="Test"
+                                //placeholder="Test"
```
Comment (tomaskikutis) [inline examples/pages/react/TagInputDocs.tsx]: don't commit commented out code :)

File: app/styles/form-elements/_tag-input.scss [hunk 3]
```diff
@@ -20,23 +21,23 @@
     width: 100%;
     border: 0;
     padding-top: 2px;
-    border-bottom: 1px solid var(--color-input-border) !important;
-    background-color: var(--color-input-bg);
+    //border-bottom: 1px solid var(--color-input-border) !important;
```
Comment (tomaskikutis) [inline app/styles/form-elements/_tag-input.scss]: remove code instead of commenting it out

File: app/styles/primereact/_pr-tag-input.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
+.p-chips-token	{
+    @include tag-label;
+    transition: background-color .2s ease-out, color .1s ease-out, opacity .1s ease-out;
+    background: $tag-label-BG-default !important;
```
Comment (tomaskikutis) [inline app/styles/primereact/_pr-tag-input.scss]: can you use more specific selectors so you don't need `!important`?

File: app-typescript/components/TagInput.tsx (hunk 1 above)
Comment (dzonidoo) [reply app-typescript/components/TagInput.tsx]: we always use the default placeholder so i thought it was necessary now... I'll remove it now...

File: examples/pages/react/TagInputDocs.tsx (hunk 2 above)
Comment (dzonidoo) [reply examples/pages/react/TagInputDocs.tsx]: agree :D

**Comment by thecalcc** [comment]
Old angular component isn't used anywhere so we're ready to merge if everything else is done.

**Review by tomaskikutis — APPROVED** [review]
fix tests before merging

**Pull request merged**